.ci/docker/common/install_gcc.sh

```diff
@@ -4,6 +4,11 @@ set -ex
 
 if [ -n "$GCC_VERSION" ]; then
 
+  if [[ "$UBUNTU_VERSION" == "22.04"* ]]; then
```

Comment:
Nit: It makes more sense to me to just add gpg-agent into the list here https://github.com/pytorch/pytorch/blob/main/.ci/docker/common/install_base.sh#L77.  If it has been installed, it should be a no-op, right?

Reply:
right, agree with you, move the gpg-agent installation into base script